dashboard/src/public/index.html

```diff
@@ -22,7 +22,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Oríon Dashboard</title>
+    <title>Orion Dashboard</title>
```

Comment:
Why did these change? Encoding issues?

Reply:
No, I thought it was a typo error. I will revert it!